.github/workflows/mutation-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: mutation-tests
+
+on:
+  push:
+    branches: [ main, release/* ]
+    tags: [ '*' ]
```

Comment:
There's no need to run these for tags - they were skipped previously as it massively slowed down the time to publish a new release.

Reply:
Sure. Updated.